lib/services/amq_event_service.ex

```diff
@@ -0,0 +1,96 @@
+defmodule Aprb.Service.AmqEventService do
+  use GenServer
+  use AMQP
+
+  alias Aprb.Service.EventService
+
+  def start_link(opts) do
+    GenServer.start_link(__MODULE__, opts, [])
+  end
+
+  def init(topic) do
+    rabbitmq_connect(topic)
+  end
+
+  defp rabbitmq_connect(topic) do
+    case Connection.open(
+      username: System.get_env("RABBITMQ_USER"),
+      password: System.get_env("RABBITMQ_PASSWORD"),
+      host: System.get_env("RABBITMQ_HOST"),
+      virtual_host: System.get_env("RABBITMQ_VHOST"),
+      heartbeat: 5,
+      ssl_options: [
+        cacertfile: System.get_env("RABBITMQ_CA_CERT_PATH"),
+        certfile: System.get_env("RABBITMQ_CLIENT_CERT_PATH"),
+        keyfile: System.get_env("RABBTIMQ_CLIENT_KEY_PATH") 
+      ]) do
```

Comment:
I have the same reaction here as I did in https://github.com/artsy/impulse/pull/166#discussion_r93941068. It would be much nicer for developers to just worry about a connection string (plus the certs, probably). Even if a connection string or URL are used for the config, then parsed to initialize this connection, I think that would be a significant improvement.

Re. where the cert files should go, I'm not sure. If it's more convenient for us to specify them as strings, maybe they could be written to a local configuration directory as part of start-up.